scripts/exglobal_atmos_sfcanl.sh

```diff
@@ -113,7 +113,8 @@ done
 # There is only a single NSST analysis at the middle of the window
 # For now use/assume it is the same at the beginning of the window if doing IAU
 if [[ "${DONST}" == "YES" ]]; then
-  cpreq "${COMIN_ATMOS_ANALYSIS}/${APREFIX}analysis.dtf.a006.nc" "${DATA}/dtfanl"
+  export NST_FILE=${NST_FILE:-${COMIN_ATMOS_ANALYSIS}/${APREFIX}analysis.dtf.a006.nc}
+  cpreq "${NST_FILE}" "${DATA}/dtfanl"
```

Comment:
It looks like there's a need to allow someone to change the NST_FILE name. You may want to add a check verifying the file exists when inside the if block prior to the cpreq (maybe cpreq does this already?). Otherwise this looks OK.